### DISCUSSION TITLE
Weblogic JMS connection

### DISCUSSION THREAD

**Discussion by michael-esmann** [jmstoolbox/jmstoolbox#176] [General]
Thx. for at great tool !!
This is only a tip:
To connect newer versions of JMSToolBox to weblogic JMS, this should be the solution: Add "--add-opens=java.base/java.io=ALL-UNNAMED" to your "JMSToolBox.ini"
Maybe add this information to https://github.com/jmstoolbox/jmstoolbox/wiki/2.9-Setup-for-Oracle-WebLogic-Server ?

**Comment by titou10titou10** [comment]
ok...
Could you copy here the error you encountered without this jvm option? 
thanks

**Reply by michael-esmann** [reply]
If I dont apply this option then JMSToolBox just hangs when I try to connect.

JMSToolBox 6.3.0, wlthint3client 12.2.1.4.0

**Comment by titou10titou10** [comment]
Done
Thanks for reporting
